src/ui/services/networkService.ts

```diff
@@ -3,12 +3,15 @@
 
 import { EventEmitter } from "events";
 import StaticService from "./staticService.ts";
-import readConfigData from "../../../ConfigReader.ts";
 
 const io = <SocketIOClientStatic>require("socket.io-client");
 var playerId;
 var pathToLolExe;
+var dataToStartGame;
 var pathToLolFolder;
+var selectedChampion;
+var championSelectStarted = false;
+var canPick = false;
 var execFile = require('child_process').execFile;
```

Comment:
`var` -> `const` or `let`. Why are these vars even here, they should be inside some instance.